File: src/main/java/leets/weeth/domain/account/domain/repository/ReceiptRepository.java
package leets.weeth.domain.account.domain.repository;

import leets.weeth.domain.account.domain.entity.Receipt;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface ReceiptRepository extends JpaRepository<Receipt, Long> {
    List<Receipt> findAllByAccountIdOrderByCreatedAtDesc(Long accountId);
}

File: src/main/kotlin/com/weeth/domain/account/domain/repository/ReceiptRepository.kt
package com.weeth.domain.account.domain.repository

import com.weeth.domain.account.domain.entity.Receipt
import org.springframework.data.jpa.repository.JpaRepository

interface ReceiptRepository : JpaRepository<Receipt, Long> {
    fun findAllByAccountIdOrderByCreatedAtDesc(accountId: Long): List<Receipt>
}
